src/interfaces.ts

```diff
@@ -17,6 +17,7 @@ export interface BridgeSettings {
  * the metadata that will be written to disk as an array of {@link Metadata}
  */
 export interface Metadata {
+	frontmatter: any;
```

Comment:
```suggestion
	frontmatter?: any;
```

Since this is optional, the key in the interface should be marked as such.